charms/trusty/clearwater-homestead/lib/config_script

```diff
@@ -23,8 +26,18 @@ local_ip=$(unit-get private-address)
 public_ip=$(unit-get public-address)
 public_hostname=homestead-$(cut -d/ -f2 <<< $JUJU_UNIT_NAME).$(config-get zone)
 node_idx=$(($(cut -d/ -f2 <<< $JUJU_UNIT_NAME) + 1))
+[ "$relation_name" != "hss" ] || [ "$(relation-get public-address)" = "" ] || hss_hostname=$(relation-get public-address)
```

Comment:
How does the public-address on the proxy charm get set to the IP of the external HSS?